allensdk/internal/core/exceptions.py

```diff
@@ -0,0 +1,22 @@
+class DataFrameKeyError(LookupError):
```

Comment:
These seem pretty handy. Probably too handy to live in internal/.

Reply:
Ah I see, maybe we can go over what exactly the `internal` folder means. I assumed that was where we should put code used by the allensdk internally, which is not necessary for end-users.

Reply:
There's [a readme](https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/internal/README.md)!

Reply:
Moved to core outside of internal folder